### PR TITLE
add informationtest.java file and added github code lines

### PR DESCRIPTION
**Pull request by rpallavisharma**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

Added InformationTest.java files. Added github code lines

### Description

Added InformationTest.java files. Added github code lines
### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Review by diemol — COMMENTED** [review]
@pallavigitwork maybe I am checking the wrong place, but the Java file is empty, is that correct?
https://github.com/pallavigitwork/seleniumhq.github.io/blob/docWork-Pallavi/examples/java/src/test/java/dev/selenium/elements/InformationTest.java#L20-L25

**Comment by rpallavisharma** [comment]
no. I added the file. let me try again.

**Comment by rpallavisharma** [comment]
please check now @diemol

File: examples/java/src/test/java/dev/selenium/elements/InformationTest.java [hunk 1]
```diff
@@ -1,7 +1,71 @@
...
+import org.openqa.selenium.By;
+import org.openqa.selenium.Rectangle;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import java.time.Duration;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class InformationTest extends BaseTest {
+public class ElementInformation {
```
Comment (diemol) [inline examples/java/src/test/java/dev/selenium/elements/InformationTest.java]: Shouldn't the class name match the file name?

Why don't you use the `BaseTest` class?

Comment (rpallavisharma) [reply examples/java/src/test/java/dev/selenium/elements/InformationTest.java]: @diemol made the change. sorry for this. that day was a bad day, when i originally created this. apologies for the inconvenience.

**Review by diemol — COMMENTED** [review]
Seems some tests are failing https://github.com/SeleniumHQ/seleniumhq.github.io/actions/runs/7697404014/job/20974440208?pr=1560#step:7:1498

**Comment by rpallavisharma** [comment]
@diemol ok- Run Java examples / test_examples (macos-latest) (pull_request) Failing after 22m  - windows passed. not sure what to do. let me know, if anything i need to do.

**Comment by diemol** [comment]
>Error:    InformationTest.informationWithElements:51 expected: <rgba(239, 239, 239, 1)> but was: <rgba(240, 240, 240, 1)>

The assertion is too strict or brittle. Maybe you can find a different assertion?

**Comment by rpallavisharma** [comment]
@diemol let me know now.

File: examples/java/src/test/java/dev/selenium/elements/InformationTest.java (hunk 1 above)
Comment (diemol) [reply examples/java/src/test/java/dev/selenium/elements/InformationTest.java]: Thank you. No need to apologize at all!

File: examples/java/src/test/java/dev/selenium/elements/InteractionTest.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 package dev.selenium.elements;
-
+import dev.selenium.BaseTest;
```
Comment (diemol) [inline examples/java/src/test/java/dev/selenium/elements/InteractionTest.java]: Adding this extra line will displace the docs that refer to this code file. If we do not need to import the `BaseTest` class, can we remove it?

**Review by diemol — CHANGES_REQUESTED** [review]

File: examples/java/src/test/java/dev/selenium/elements/InteractionTest.java (hunk 2 above)
Comment (rpallavisharma) [reply examples/java/src/test/java/dev/selenium/elements/InteractionTest.java]: removed. let me know if anything else required.

Comment (diemol) [reply examples/java/src/test/java/dev/selenium/elements/InteractionTest.java]: Thank you!

Comment (rpallavisharma) [reply examples/java/src/test/java/dev/selenium/elements/InteractionTest.java]: :) thank you Diego.

**Review by diemol — APPROVED** [review]
Thank you, @pallavigitwork!
